src/components/mdx/HeaderCard.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+const HeaderCard = ({header, children}) => {
```

Comment:
**suggestion:** Couldn't "Header Card" be just "Card" with optional header prop or something?